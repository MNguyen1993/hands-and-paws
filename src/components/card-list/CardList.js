import React from 'react';
import Card from '../card/Card';

const CardList = ({ pets }) => {
	return (
		<div>
			{pets.map((pet, i) => {
				return (
					<Card key={i} id={pet.id} name={pet.name} email={pet.email} />
				);
			})}
		</div>
	);
};

export default CardList;
