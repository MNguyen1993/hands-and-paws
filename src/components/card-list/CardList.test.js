import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('renders CardList component', () => {
	const mockPets = [
		{
			id: 1,
			name: 'John Snow',
			email: 'johnsnow@gmail.com',
		},
	];

	expect(shallow(<CardList pets={mockPets} />)).toMatchSnapshot();
});
