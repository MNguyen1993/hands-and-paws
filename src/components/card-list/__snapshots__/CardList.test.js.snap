// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CardList component 1`] = `
<div>
  <Card
    email="johnsnow@gmail.com"
    id={1}
    key="0"
    name="John Snow"
  />
</div>
`;
